data/coalition missions.txt

```diff
@@ -60,12 +60,19 @@ mission "Coalition: First Contact"
 			label oppressive
 			`	The spider says, "In separate boxes, they each species keep. And in subtle ways, your development and discoveries they limit. And on your technology, restrictions they place."`
 			`	The beetle says, "Or, wrong are we? And the jump drive that brought you here, a gift from the Quarg it is? They its secret no longer hoard?" You tell it that the Quarg do indeed refuse to give jump drives to humans, and it says, "Then, prisoners indeed you are."`
-			label next
+			branch next
```

Comment:
No. Look at `Wanderers: Alpha Surveillance I`. `branch know` in that case literally acts as another `label know`, except with a condition. It then bypasses the original `label know` if the condition is fulfilled; otherwise, the original `label know` is used.